Xwt.XamMac/Xwt.Mac/TextLayoutBackendHandler.cs

```diff
@@ -260,8 +260,8 @@ public void Draw (CGContext ctx, CGColor foregroundColor, double x, double y)
 					TextLayout.AddTextContainer (TextContainer);
 					TextStorage.AddLayoutManager (TextLayout);
 
-					TextLayout.DrawBackgroundForGlyphRange (new NSRange(0, Text.Length), new CGPoint (x, y));
-					TextLayout.DrawGlyphsForGlyphRange (new NSRange(0, Text.Length), new CGPoint (x, y));
+					TextLayout.DrawBackground (new NSRange(0, Text.Length), new CGPoint (x, y));
+					TextLayout.DrawBackground (new NSRange(0, Text.Length), new CGPoint (x, y));
```

Comment:
This should be `DrawGlyphs`, it's the cause of https://github.com/xamarin/vsmac/pull/5897#issuecomment-1043236008.

Reply:
Fixed.